pkg/kyma/docs/install.md

```diff
@@ -0,0 +1,108 @@
+# install
+
+## Description
+
+Installs Kyma on a running Kubernetes cluster.
+
```

Comment:
`This command installs`...? or `Use it to install...`, then we have a full sentence :D